package.json

```diff
@@ -52,6 +52,7 @@
     "lodash": "4.17.21",
     "ms": "2.1.3",
     "multiformats": "9.6.4",
+    "node-cache": "5.1.2",
```

Comment:
`node-cache` may not be supported in the browser. Though when I went and looked at it's dependencies, i can't see why it wouldn't work in the browser? https://github.com/isaacs/node-lru-cache seems to have explicit browser support.

More generally, how'd you go about deciding on a TTL cache vs. other types of caches (e.g. LRU)

Reply:
In context of DID resolution, there must be an element of TTL as documents will be changed overtime. But will switched to the lib you referenced because:

1. It has 10x the usage
1. Has support for TTL
1. Seems to have explicit support. (Though I believe `node-cache` made a fix to make it browser compatible and tests pass in `test:browser` mode).

Thanks for the great find!